docs/source/dialogue_manager.rst

```diff
@@ -0,0 +1,39 @@
+Dialogue Manager
+================
+
+The :py:class:`DialogueManager <moviebot.dialogue_manager.dialogue_manager>` tracks the dialogue state and decides what action the system should take.
+It comprises a dialogue state tracker and a dialogue policy.
+
+
+Dialogue State Tracker
+----------------------
+
+This component is responsible for keeping track of the dialogue state and dialogue context based on the dialogue acts of the agent and the user.
+In the current implementation, :py:class:`DialogueStateTracker <moviebot.dialogue_manager.dialogue_state_tracker>` is rule-based.
+
+Dialogue Policy
+---------------
+
+This component decides what is the next action of the system based on the current dialogue state, i.e., it generates the next dialogue act of the system.
+It defines the flow of the conversation, i.e., what steps an agent must take at every stage. 
+MovieBot supports two types of dialogue policies: rule-based and neural-based.
+
+Rule-based Dialogue Policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Based on a set of pre-defined rules and the current dialogue state, this policy decides the next action of the system.
```

Comment:
Sounds repetitive. Perhaps simplify to "The next system action is determined based on a set of pre-defined rules, given the current dialogue state."

Reply:
Replaced by suggestion.